packages/react/src/file-upload/FileUpload.tsx

```diff
@@ -261,6 +270,10 @@ export const FileUpload = forwardRef<HTMLInputElement, FileUploadProps>(
 				{(a11yProps) => {
 					return (
 						<Stack gap={1.5}>
+							<div css={visuallyHiddenStyles} role="status">
```

Comment:
I can't get VO to announce the status on adding files either, and I think it's because the button gets focused so it gets overridden as it's less important. Changing to `alert` seems to announce it though

Reply:
Urgh, even alerts aren't being announced when adding files for me now! Remove always works ok. Another issue which they haven't cared about is where focus goes when you remove. Tags we were told to move it. I don't think we should here, but it's interesting that they didn't care…